app/components/primer/beta/auto_complete/item.html.erb

```diff
@@ -0,0 +1,14 @@
+<%= render Primer::BaseComponent.new(**@system_arguments) do %>
+  <span class="ActionList-content">
+    <% if leading_visual %>
+        <span class="ActionList-item-visual ActionList-item-visual--leading">
+            <%= leading_visual %>
+        </span>
+    <% end %>
+    <span class="ActionList-item-label"><%= content %></span>
```

Comment:
@joelhawksley I added a template for Item so I could add leading/trailing visuals. Is this `content` slot okay? If we make it a new `label` prop it might make integrating this with dotcom more difficult, but I'm open to that. I'm not really sure what `content` is/does.

Reply:
This looks good to me!